### PR TITLE
Move TikzGraphs require to Julia 0.4 or later

### PR DESCRIPTION
**Pull request by mykelk**
Earlier PR did not increment the minor version. This one should have it right now.

### CODE REVIEW

**Comment by tkelman** [comment]
probably best to remove the files related to the 0.1.2 and 0.1.3 tags here

File: TikzGraphs/versions/0.2.0/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.4
+TikzPictures
+LightGraphs
+Compat
```
Comment (tkelman) [inline TikzGraphs/versions/0.2.0/requires]: also a good idea to declare a minimum version dependency that implements the features you need to use

Comment (mykelk) [inline TikzGraphs/versions/0.2.0/requires]: I don't think there is a specific version dependency (i.e., this update doesn't depend on any new features of these packages). Would it be bad practice to just list the most recent version numbers for TikzPictures, LightGraphs, and Compat?

Comment (mykelk) [inline TikzGraphs/versions/0.2.0/requires]: Getting rid of the v0.1.2 should resolve the Travis issue.

Comment (tkelman) [inline TikzGraphs/versions/0.2.0/requires]: using the most recent version would be too tight, but maybe better that way than too loose? whatever you've used Compat for to fix deprecations has probably not been implemented in that package forever.

**Pull request closed without merging**